sis-local/example/README.md

```diff
@@ -0,0 +1,12 @@
+# Example usage of streamlit_in_snowflake
+
+1. Create a secrets.toml file in this repo, as described in the parent directory README
+1. Create and activate a python 3.8 virtual environment
+1. Run the following
```

Comment:
```suggestion
1. Create a secrets.toml file in this repo, as described in the parent directory README
2. Create and activate a python 3.8 virtual environment
3. Run the following
```

Reply:
It is pretty neat, though, that markdown previews will automatically fix the numbering for you :)